src/configuration.rs

```diff
@@ -169,6 +167,11 @@ impl PatternExpression {
             }
             Some(attribute_bytes) => attribute_bytes,
         };
+
+        // if someone would have the P_E be:
```

Comment:
I do not get this comment. For backwards compat?

why not :fire: to `PatternExpressions`?

Reply:
Yes! We should burn them as soon as the kuardrant operator doesn't use them to configure this wasm filter anymore...

Reply:
yeah, that would allow us to merge this without waiting for operator to implement that.